src/ProgramBlockEditor.js

```diff
@@ -523,11 +589,17 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
 
     getThemedCharacter() {
         if (this.props.theme === 'space') {
-            return <SpaceShipIcon className='ProgramBlockEditor__character-column-character' />
+            return <SpaceShipIcon
+                transform={`rotate(${this.props.characterState.getDirectionDegrees() - 90} 0 0)`}
```

Comment:
In my own work in this area, I thought I had seen a variation where the transformation was defined as a variable and reused.  That seems more appropriate.

Reply:
Good idea, I used a const for the transform.